lib/stripe/stripe_client.rb

```diff
@@ -110,7 +110,7 @@ def self.default_connection_manager(config = Stripe.config)
     # both socket errors that may represent an intermittent problem and some
     # special HTTP statuses.
     def self.should_retry?(error,
-                           method:, num_retries:, config: Stripe.config)
```

Comment:
Nit: I think since this method is defined with `def self.` and not `private def`, removing `method:` from the signature is breaking and should be mentioned in the changelog?

I'm not sure whether our release automation picks up `## Changelog` entries from PR description for `sdk-release/next-major` like it does with master/beta branch releases, but I would edit the PR description to contain something like this so we don't forget
```
## Changelog
* When no `x-stripe-should-retry` header is set, the library now retries all requests with `status >= 500`, not just specific non-POST statuses.
* Remove `method` parameter from `StripeClient.should_retry?`
```

Reply:
LGTM!